website/server/controllers/api-v3/tasks.js

```diff
@@ -287,7 +287,7 @@ api.getUserTasks = {
   method: 'GET',
   url: '/tasks/user',
   middlewares: [authWithHeaders({
-    userFieldsToExclude: ['inbox'],
+    userFieldsToInclude: ['_id'],
```

Comment:
Just checking... Getting tasks involves ordering them based on user.tasksOrder and also calls `setNextDue` which uses user.preferences for date-related fields. This new limitation won't affect that will it?